schema/types.go

```diff
@@ -77,3 +78,12 @@ type PodInfo struct {
 type ContainerInfo struct {
 	InfoType
 }
+
+// Supported metric names, used as keys for all map[string]*store.TimeStore
+const cpuLimit = "cpu/limit"
```

Comment:
How about a generic resource struct { usage int, limit int } and have one such struct for every resource type? Essentially we will have a static struct with all the metrics defined. Ok to tackle in a future PR.